Sources/View/UICollectionView+Reusable.swift

```diff
@@ -128,3 +290,44 @@ public extension UICollectionView {
       return typedView
   }
 }
+
+private var registeredCellTypesKey: Void?
+
+private extension UICollectionView {
+  private class RegisteredCellTypes {
+    var cellReuseIdentifiers: Set<String>
+    var supplementaryViewReuseIdentifiers: [String: String] //[ElementKind: ReuseIdentifier]
+    init() {
```

Comment:
Might as well use `=` directly on the properties declaration lines and avoid having to implement `init` there 😉